package.json

```diff
@@ -147,45 +140,43 @@
         "category": "Buf",
         "icon": "$(debug-stop)",
         "title": "Stop Buf Language Server"
-      },
-      {
-        "command": "buf.update",
-        "category": "Buf",
-        "icon": "$(sync)",
-        "title": "Update CLI",
-        "description": "Check for updates and install the latest version of the Buf CLI."
       }
     ],
     "configuration": {
       "title": "Buf",
       "properties": {
         "buf.commandLine.path": {
           "type": "string",
-          "description": "The path to a specific install of Buf to use. Relative paths are supported and are relative to the workspace root."
+          "description": "The path to a specific install of Buf to use. Relative paths are supported and are relative to the workspace root.",
+          "deprecationMessage": "Deprecated: Buf path is no longer configurable."
         },
         "buf.commandLine.version": {
           "type": "string",
-          "description": "Specific version (git tag e.g. 'v1.53.0') of Buf release to download and install."
+          "description": "Specific version (git tag e.g. 'v1.53.0') of Buf release to download and install.",
+          "deprecationMessage": "Deprecated: Buf version is no longer configurable."
         },
         "buf.restartAfterCrash": {
```

Comment:
Almost all of these options were supposed to be completely deleted.

Reply:
We want to make sure that the extension continues to be compatible with any existing `settings.json` files that users may have checked into their projects and/or configured locally. Deprecating them removes them from the settings page of the extension and shows the deprecation notice when editing the `settings.json` file in VS Code.